Source/WebCore/rendering/RenderTheme.cpp

```diff
@@ -1418,37 +1311,122 @@ bool RenderTheme::hasListButtonPressed(const RenderObject& renderer) const
 }
 #endif
 
-#if PLATFORM(IOS_FAMILY)
+// FIXME: iOS does not use this so arguably this should be better abstracted. Or maybe we should
+// investigate if we can bring the various ports closer together.
+void RenderTheme::adjustButtonOrCheckboxOrColorWellOrInnerSpinButtonOrRadioOrSwitchStyle(RenderStyle& style, const Element* element) const
```

Comment:
Not a big fan of the method name, but don't have better suggestions, so I'm fine with this 😅 

I've used "button-like" control to describe a similar concept on iOS, but that also includes `<select>`.

Reply:
It seems okay to me for it to have an unattractive name as we should make it disappear or make it cross-platform at which point we can rename.